pom.xml

```diff
@@ -82,6 +82,7 @@
         <version.lib.awaitility>3.1.6</version.lib.awaitility>
         <version.lib.weld-junit>3.0.0.Final</version.lib.weld-junit>
         <version.lib.jmh>1.23</version.lib.jmh>
+        <version.lib.vertx-core>4.3.7</version.lib.vertx-core>
```

Comment:
FYI, there is now a 4.3.8, but that's not a required change. This test dependency is approved.

Reply:
thx! upgraded